tools/ci_build/github/azure-pipelines/templates/win-wasm-ci.yml

```diff
@@ -29,7 +29,7 @@ jobs:
   variables:
     EnvSetupScript: setup_env.bat
```

Comment:
BTW, this bat is only for C# tests. Normal C/C++ or python build do not need this.